packages/@okta/vuepress-site/docs/guides/device-context/main/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Device Context Overview
+---
+
+<div class="oie-embedded-sdk">
+
+<ApiLifecycle access="ie" /><br>
+
+Enable a new device rule for your Org's authentication policy.
+
+---
+**Learning outcomes**
+
+* Send a unique device ID to the Okta servers with a custom HTTP request header.
+* Understand how device context enables Adaptive Multifactor Authentication (AMFA).
+
+**What you need**
+
+* <StackSnippet snippet="whatyouneedsdk" />
+* <StackSnippet snippet="whatyouneedorg" />
+
+**Sample code**
+
+* <StackSnippet snippet="samplecode" />
+
+---
+
+## Overview
+
+Server-side web applications can use the `X-Device-Token` custom HTTP request header to send a user's device ID to an Okta org. The ID:
+
+* Is 32 characters or less.
+* Identifies the specific user device.
+* Is unique across all devices.
+
+For server-side apps using an embedded SDK, developers must create the ID and attach it to the `X-Device-Token` header. The ID informs two features within an Org that flag "a request is coming to the Org from a new device":
+
+* [Device Context](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/guides/devices/devcontext-main.htm)
+* [Behavior Detection](https://help.okta.com/oie/en-us/Content/Topics/Security/behavior-detection/about-behavior-detection.htm)
+
+Administrators can write authentication policy rules for sign-in requests from a new device using the **new device behavior**. The following diagram shows this flow:
+
+<div class="full">
+
+![Flow diagram showing how the new device behavior works](/img/advanced-use-cases/device-context-new-behavior-flow.png)
+
+</div>
+
+> **Note:** Find your Org's definition of **New Device** in your Admin Console under **Security** > **Behavior Detection**.
+
+## Update your application to send an X-Device-Token header
+
+<StackSnippet snippet="integrationsteps" />
+
+## An Example: Adaptive Multi-Factor Authentication
+
+An Adaptive Multi-Factor Authentication (AMFA) policy means that users are prompted for a different number of authentication factors based on a number of criteria, including:
+
```

Comment:
```suggestion
Users with an Adaptive Multi-Factor Authentication (AMFA) policy are prompted for a different number of authentication factors based on a number of criteria, including:

```

Reply:
Changed to "Apps with an Adaptive Multi-Factor Authentication (AMFA) policy prompt users for..."  to remove the passive verb.